Makefile

```diff
@@ -0,0 +1,47 @@
+NAME=cimgui-go Code Generator
+
+.PHONY: all
+
+## all: generates both bindings.
+all: generate
+
+.PHONY: setup
+## setup: pulls down dependencies.
+setup:
+	go get -v -d ./...
+
+## _gencode: is an internal rule. It generates binding basing on env variables set in imgui and implot rules
+_generate: setup
```

Comment:
Looks like this needs to be a function [see example here](https://coderwall.com/p/cezf6g/define-your-own-function-in-a-makefile), then you'll be able to write cimplot/cimgui targets easier: they'll just set set needed variables and then call the function

Reply:
Oh, I can't imagine how did i miss it while looking around makefile's documentation...
Thank you very much!

Reply:
> and maybe in th efuture, if we'll need to use go generate or something like that, we may put it there.

also, I'm wondering if we shouldn't obtain `gofmt` here? User isn't obligated to have it installed tbh.